### DISCUSSION TITLE
BitLocker

### DISCUSSION THREAD

**Discussion by mecinformatica** [sepinf-inc/IPED#1697] [Ideas]
Pelas minhas pesquisas sobre esta excelente ferramenta, o IPED, não é possível index uma imagem de uma unidade que esteja com BitLocker ativo, correto? Seria possível, em versões futuras, colocar esta funcionalidade? Possibilitando colocar a chave do BitLocker e a própria ferramenta já abrir a unidade criptografada...

**Comment by lfcnassif** [comment]
Unfortunately this must be implemented at a lower level, in the Sleuthkit project (https://github.com/sleuthkit/sleuthkit), the library used by us to decode file systems. I just found a very old related PR there (https://github.com/sleuthkit/sleuthkit/pull/655) that seems far away from even a draft... You can ask the feature to Sleuthkit project and, as an open source project, like us, help them to implement.

PS: If BitLocker support is implemented into Sleuthkit, we can integrate it somewhat easily.

**Reply by mecinformatica** [reply]
Certo, vou avançar com este assunto. Obrigado por esclarecer.
